api/swagger.yaml

```diff
@@ -7338,12 +7338,12 @@ paths:
       parameters:
         - name: "since"
           in: "query"
-          description: "Show events created since this timestamp then stream new events."
-          type: "string"
+          description: "Show events created since this timestamp then stream new events.  This is in epoch time."
```

Comment:
Perhaps use the same wording (`UNIX timestamp`) as for container logs; https://github.com/moby/moby/blob/5c152ea10f6ab545471d07af49bad4f870e08f0b/api/swagger.yaml#L5182-L5191

Reply:
Actually, looks like some tweaking of the wording may be needed, as I think it supports timestamps with nanosecond precision; https://github.com/moby/moby/blob/8e610b2b55bfd1bfa9436ab110d311f5e8a74dcb/api/types/time/timestamp.go#L98-L129